client/payments-welcome/faq.tsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import { Icon, help } from '@wordpress/icons';
+import { Panel, PanelBody as PanelBodyBase } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import strings from './strings';
+
+const PanelBody: React.FC< PanelBodyBase.Props > = ( props ) => {
+	return <PanelBodyBase initialOpen={ false } { ...props } />;
+};
+
+const FrequentlyAskedQuestions: React.FC = () => {
+	// const [panelOpen, setPanelOpen] = useState(false);
```

Comment:
https://github.com/Automattic/wc-calypso-bridge/blob/master/src/payments-welcome/faq.tsx

Looks like we don't use this, can we just delete this line?

Reply:
Thank you! Nice catch 👍 Change pushed to https://github.com/woocommerce/woocommerce-admin/pull/8083/commits/a75929a1ffd9af4132ab4aa0417946a67336ccfc